src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -30,6 +30,7 @@ import {
 interface Props {
   artist: ArtworkFilter_artist
   hideTopBorder?: boolean
+  filterState?: FilterState
```

Comment:
Updated the following bits so that if in the future we need to connect to that `showZeroState` toggle we easily can.